ArduPlane/sensors.cpp

```diff
@@ -215,6 +219,10 @@ void Plane::update_sensor_status_flags(void)
         control_sensors_enabled |= MAV_SYS_STATUS_LOGGING;
     }
 
+    if (g.fs_batt_voltage != 0 || g.fs_batt_mah != 0) {
```

Comment:
On reflection we do it as if current < fs_batt_voltage so that doesn't matter. Will change to > 0